components/member-card/card.module.scss

```diff
@@ -49,7 +49,7 @@
   margin: 15px 0;
 }
 .nameOfPersonForNewMember {
-  @extend.nameOfPerson;
+  @extend.nameofperson !optional;
```

Comment:
Could you provide more information behind adding !optional here , please

Reply:
Linting gave me the suggestion, if not done this was giving long errors in the console, but once again I'll look into it. 👍🏼

Reply:
My bad here, somehow the `name-casing` changed and `@extend` was not successful.
Made the changes.